docs/translations-needed.md

```diff
@@ -13,6 +13,7 @@ This default value is hardcoded as a default to the localize method in the SeekB
 ## Status of translations
 
 <!-- START langtable -->
+
```

Comment:
The check-translations task would remove these lines when it updates the table.

Reply:
yeah, we should probably fix that up since remark adds these lines and we'll get thrashing, but yeah, I should just remove these lines.